src/src/modules/auth/auth.service.ts

```diff
@@ -123,7 +123,17 @@ export class AuthService {
               }`,
 		};
 		const userRes = await this.hasuraService.postData(query);
-		console.log('userRes', userRes);
+
+		return userRes;
+	}
+
+	public async resetPasswordUsingOtp(req, response) {
+		console.log('req', req);
+		const username = req.username;
+		const hash = req.hash;
+		const otp = req.otp;
+		const reason = req.reason;
+		const userRes = await this.getUserByusername(req);
```

Comment:
not using renamed name?

Reply:
same here we created new function getUserByusername and using that